src/rabbit_amqqueue_process.erl

```diff
@@ -998,13 +1023,15 @@ i(effective_policy_definition,  #q{q = Q}) ->
         undefined -> [];
         Def       -> Def
     end;
-i(exclusive_consumer_pid, #q{exclusive_consumer = none}) ->
+i(exclusive_consumer_pid, #q{active_consumer = none}) ->
```

Comment:
I understand that single active consumer feature is reusing the field for exclusive consumer, but should we reuse the info item? If there is a single ective consumer it will be reported as exclusive, which is close, but can be confusing.

Reply:
This is fixed, single active consumer don't use exclusive consumer info items anymore.